docs/extensions/datacollector-migration.md

```diff
@@ -0,0 +1,44 @@
+# Migrate your TestPlatform v0(TPv0)/TestPlatoformv1(TPv1) datacollector to TestPlatform v2(current version) datacollector
+DataCollectors were supported in TPv0(mstest.exe)/TPv1(vstest.console.exe). 
+This document will walk you through the changes that are required to migrate your TPv0/TPv1 DataCollector to work with TPv2.
+
+## Referencing DataCollector Framework
+Previously, `DataCollector` absract class was present in `Microsoft.VisualStudio.QualityTools.ExecutionCommon.dll` under namespace `Microsoft.VisualStudio.TestTools.Execution`.
+
+Now, `DataCollector`abstract class is present in Object Model. Add reference to `Microsoft.TestPlatform.ObjectModel` (preview) nuget package. DataCollector APIs are present under namespace `Microsoft.VisualStudio.TestPlatform.ObjectModel.DataCollection`
+
+## Deprecated Attributes
+Currently `DataCollectorFriendlyNameAttribute` and `DataCollectorTypeUriAttribute` are the only attirbutes that are supported. 
+Following attributes have been deprecated and should be removed if already being used.
```

Comment:
There is no alternative as such.
In TestPlatform, we have realized that these are no more required.
Updated doc with the same comment.